sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -4113,6 +4113,7 @@ _ZN4sycl3_V17handler12addReductionERKSt10shared_ptrIKvE
 _ZN4sycl3_V17handler13getKernelNameB5cxx11Ev
 _ZN4sycl3_V17handler15ext_oneapi_copyENS0_3ext6oneapi12experimental16image_mem_handleENS0_5rangeILi3EEERKNS4_16image_descriptorEPvS7_S7_S7_
 _ZN4sycl3_V17handler15ext_oneapi_copyENS0_3ext6oneapi12experimental16image_mem_handleEPvRKNS4_16image_descriptorE
+_ZN4sycl3_V17handler15ext_oneapi_copyENS0_3ext6oneapi12experimental16image_mem_handleES5_RKNS4_16image_descriptorE
```

Comment:
Are you not missing some symbols here? In `sycl_symbols_windows.dump` you added four new symbols to check. But here only one has been added.